Libplanet/Net/ITransport.cs

```diff
@@ -1,49 +1,162 @@
+        /// <returns>An awaitable task without value.</returns>
         Task StopAsync(
             TimeSpan waitFor,
             CancellationToken cancellationToken = default(CancellationToken));
 
+        /// <summary>
+        /// Conduct peer discovery for given <paramref name="bootstrapPeers"/>.
+        /// </summary>
+        /// <param name="bootstrapPeers">A <see cref="IEnumerable{T}"/> of <see cref="Peer"/>s
+        /// to bootstrap.</param>
+        /// <param name="pingSeedTimeout">A timeout of waiting for the reply of <see cref="Ping"/>
+        /// message sent to seed <see cref="Peer"/>.
+        /// If <c>null</c> is given, task never halts by itself
+        /// even the target seed gives no any response.</param>
+        /// <param name="findNeighborsTimeout">A timeout of waiting for the reply of
+        /// <see cref="FindNeighbors"/> message sent to seed <see cref="Peer"/>.
+        /// If <c>null</c> is given, task never halts by itself
+        /// even the target seed gives no any response.</param>
+        /// <param name="depth">Recursive operation depth to search peers from network.</param>
+        /// <param name="cancellationToken">
+        /// A cancellation token used to propagate notification that this
+        /// operation should be canceled.</param>
+        /// <returns>An awaitable task without value.</returns>
         Task BootstrapAsync(
             IEnumerable<BoundPeer> bootstrapPeers,
             TimeSpan? pingSeedTimeout,
             TimeSpan? findNeighborsTimeout,
             int depth,
             CancellationToken cancellationToken);
 
+        /// <summary>
+        /// Sends the <paramref name="message"/> to given <paramref name="peer"/>.
+        /// </summary>
+        /// <param name="peer">A <see cref="Peer"/> to send message to.</param>
+        /// <param name="message">A <see cref="Message"/> to send.</param>
+        /// <param name="cancellationToken">
+        /// A cancellation token used to propagate notification that this
+        /// operation should be canceled.</param>
+        /// <returns>An awaitable task without value.</returns>
+        Task SendMessageAsync(BoundPeer peer, Message message, CancellationToken cancellationToken);
+
+        /// <summary>
+        /// Sends the <paramref name="message"/>
+        /// to given <paramref name="peer"/> and wait for its single reply.
+        /// </summary>
+        /// <param name="peer">A <see cref="Peer"/> to send message to.</param>
+        /// <param name="message">A <see cref="Message"/> to send.</param>
+        /// <param name="timeout">A timeout of waiting for the reply of the message.</param>
+        /// <param name="cancellationToken">
+        /// A cancellation token used to propagate notification that this
+        /// operation should be canceled.</param>
+        /// <returns>Reply of the <paramref name="message"/>
+        /// sent from <paramref name="peer"/>.</returns>
```

Comment:
I'm not sure between `sent from` and `sent by`. 🤔 

```suggestion
        /// <returns>The replies of the <paramref name="message"/>
        /// sent by <paramref name="peer"/>.</returns>
```